Neos.Neos/Classes/Command/SiteCommandController.php

```diff
@@ -111,31 +156,93 @@ public function createCommand($name, $packageKey, $nodeType, $nodeName = null, $
     }
 
     /**
-     * Remove site with content and related data (with globbing)
+     * Import sites
      *
-     * In the future we need some more sophisticated cleanup.
+     * This command allows importing sites from the given path/package. The format must
+     * be identical to that produced by the exportAll command.
      *
-     * @param string $siteNode Name for site root nodes to clear only content of this sites (globbing is supported)
+     * !!! The live workspace has to be empty prior to importing. !!!
```

Comment:
```suggestion
     * The live workspace has to be empty prior to importing.
```

a bit too loud i cannot hear anymore.